src/main/java/seedu/address/logic/commands/grade/GradeEditCommand.java

```diff
@@ -106,8 +106,8 @@ public boolean equals(Object other) {
     }
 
     /**
-     * Stores the details to edit the person with. Each non-empty field value will replace the
-     * corresponding field value of the person.
+     * Stores the details to edit the grade with. The non-empty field value will replace the
+     * field value of the grade.
```

Comment:
Remove "The non-empty field value will replace the field value of the grade." Irrelevant here since `Grade` is an enum